lib/data_services_api/sapint_converter.rb

```diff
@@ -0,0 +1,147 @@
+# frozen_string_literal: true
+
+module DataServicesApi
+  class SapiNTConverter
+    def initialize(dsapi_query)
+      @dsapi_query = JSON.parse(dsapi_query)
+    end
+
+    # Converts a DSAPI query to SAPINT query
+    def to_sapint_query
```

Comment:
I thought after our discussion on Thursday you were going to leave the URL parameters as a `Hash`, so that Faraday would handle them correctly? Or is `to_sapint_query` not used any more, in which case it should be deleted